boards/msb-430/include/board.h

```diff
@@ -43,8 +43,8 @@ extern "C" {
 /**
  * @brief   Define the CPU model for the <msp430.h>
  */
-#ifndef __MSP430F1612__
-#define __MSP430F1612__
+#ifndef MSP430F1612
```

Comment:
No, most of them are in fact correct, just the `__MSP430F...` and so on are incorrect. Also, to rework a PR you don't need to close it. Just do either `git commit --amend` if you want to change your last commit or use `git rebase -i upstream/master` if you want to change an older commit.